src/components/Date/DateField.js

```diff
@@ -11,20 +11,28 @@ import { isToday, isYesterday } from '../../util/datetime';
 
 type Props = {
     date: string,
+    dateFormat?: Object,
+    includeCommas?: boolean,
```

Comment:
Matching parity meant including the comma so I added it as an option.

Reply:
invert it, by default let it return commas, and change the other places to say omitCommas